<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanResults</class>
 <widget class="QDialog" name="ScanResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="9">
    <widget class="QTreeWidget" name="scanResultsWidget">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>SSID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BSSID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Signal</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Freq</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Flags</string>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true" extracomment="No need to translate">Add Network</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="wpsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>WPS Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="chooseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>76</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="filterField">
     <property name="placeholderText">
      <string>Filter SSID</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>76</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="scanButton">
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
